plugins/modules/rds_cluster.py

```diff
@@ -90,13 +90,18 @@
     backup_retention_period:
         description:
           - The number of days for which automated backups are retained (must be within V(1) to V(35)).
-            May be used when creating a new cluster, when restoring from S3, or when modifying a cluster.
-        type: int
+          May be used when creating a new cluster, when restoring from S3, or when modifying a cluster.
+          type: int
```

Comment:
This change is likely what's causing the docs error. Reverting it should fix it:
```suggestion
            May be used when creating a new cluster, when restoring from S3, or when modifying a cluster.
        type: int
```